.github/workflows/publish.yaml

```diff
@@ -4,6 +4,18 @@ on:
   release:
     types:
       - published
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: "Branch or tag to publish"
+        required: false
+        type: string
+        default: "main"
```

Comment:
Not really required since the workflow dispatch already prompts for the branch/tag to run against.

```suggestion
```